src/Authentication/LoginHandler.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Bugcache\Authentication;
+
+use Aerys\Bootable;
+use Aerys\Logger;
+use Aerys\ParsedBody;
+use Aerys\Request;
+use Aerys\Response;
+use Aerys\Server;
+use Aerys\ServerObserver;
+use Aerys\Session;
+use Amp\Promise;
+use Amp\Success;
+use Bugcache\Mustache;
+use Bugcache\SessionKeys;
+use Bugcache\Storage\AuthenticationRepository;
+use Bugcache\Storage\UserRepository;
+use function Aerys\parseBody;
+use function Amp\resolve;
+
```

Comment:
Usually, you should know where a class comes from without looking at it's origin. Maybe we should just always import just the namespace like I do for MySQL now: `use Amp\Mysql` and then `Mysql\Pool`. Way better than group statements for readability.